esp-hal-common/src/soc/esp32c2/peripherals.rs

```diff
@@ -20,9 +20,11 @@ pub(crate) use self::peripherals::*;
 // peripheral (no `PSRAM`, `RADIO`, etc. peripheral in the PACs), so we're
 // creating "virtual peripherals" for them.
 crate::peripherals! {
+    ADC1 <= virtual,
     APB_CTRL <= APB_CTRL,
     APB_SARADC <= APB_SARADC,
```

Comment:
I don't have too much context here, but should we be removing APB_SARADC etc now that we have added virtual new ones?